### ISSUE TITLE
Doesnt work at all.

### ISSUE THREAD

**Issue by Abel6669** [Prabh-Saini/inconspicuousname#61]
Cool idea but broken

**Comment by Prabh-Saini** [comment]
yeah there are many glitches and crap that just doesn't work on other peoples computers. theres massive issues in the code at the moment like mobile not automatically logging in, desktop sometimes not logging in, and no logs. but this is my first real python project and I hope to fix these issues, especially before vacation haha. also busy with school work and also just to lazy to work on it when i can play msfs 2020 :)

**Comment by Prabh-Saini** [comment]
hi there, i have a new [version](https://github.com/Prabh-Saini/inconspicuousname/releases/tag/v0.9.6) which i did some changes to. if you want, you can check it out. i think it should've fix a lot of the issues but if there is an issue please put in the error logs, thanks.
